src/EnergyPlus/DataRuntimeLanguage.cc

```diff
@@ -108,7 +108,7 @@ namespace DataRuntimeLanguage {
     int const OperatorGreaterThan(12);    // >
     int const OperatorRaiseToPower(13);   // ^
     int const OperatorLogicalAND(14);     // &&
-    int const OperatiorLogicalOR(15);     // ||
+    int const OperatorLogicalOR(15);     // ||
```

Comment:
I dunno.  I kinda liked Operatior.  And it's been in there for 11 years.  OK fine.